src/lib/Advertiser.ts

```diff
@@ -626,21 +645,50 @@ export class ResolvedAdvertiser extends EventEmitter implements Advertiser {
         return false;
       }
 
+      try {
+        const mdnsStatus = await this.resolvedInvoke(
+          bus,
+          "org.freedesktop.DBus.Properties",
+          "Get",
+          {
+            body: ["org.freedesktop.resolve1.Manager", "MulticastDNS"],
+            signature: "ss",
+          },
+        );
+
+        if (mdnsStatus[0].type !== "s") {
```

Comment:
When I execute the advertiser with a default configuration `mdnsStatus` has the following value for me:
`[ [ { type: 's', child: [] } ], [ 'no' ] ]``. So wouldn't the check need to be corrected like the following:?

```suggestion
        if (mdnsStatus[0][0].type !== "s") {
```

Reply:
Probably, yeah. I'll have this PR revised sometime today.

Reply:
I think the PR is already in a good state. We could maybe improve on presenting/rendering the error object (currently it's just json serialized). The only big thing missing is the wiki post.
Other than that I'm very happy with including the PR into the beta branch 🎉 

Thanks a lot for the work you put into 🚀